components/Fields/Stars.vue

```diff
@@ -20,7 +20,7 @@ export default defineNuxtComponent({
 <template>
   <div class="stars-data tw-text-amber-500 tw-mb-1">
     <FontAwesomeIcon
-      v-for="index in new Array(stars)"
+      v-for="index in Number(stars)"
```

Comment:
yes in props it's a number, but it doesn't infer the data. Vue shows a warning because of wrong data type send through props but it's not blocking rendering of Stars component. That's why we have to force casting within template (which is not the best).

Reply:
Can you fix the PR to just remove the Array, please